src/content_sao.cpp

```diff
@@ -541,12 +546,25 @@ void LuaEntitySAO::setAnimation(v2f frame_range, float frame_speed, float frame_
 	m_animation_sent = false;
 }
 
+void LuaEntitySAO::getAnimation(v2f *frame_range, float *frame_speed, float *frame_blend)
+{
+	*frame_range = m_animation_range;
+	*frame_speed = m_animation_speed;
+	*frame_blend = m_animation_blend;
+}
+
 void LuaEntitySAO::setBonePosition(std::string bone, v3f position, v3f rotation)
 {
 	m_bone_position[bone] = core::vector2d<v3f>(position, rotation);
 	m_bone_position_sent = false;
 }
 
+void LuaEntitySAO::getBonePosition(std::string bone, v3f *position, v3f *rotation)
```

Comment:
It doesn't work for me. I tried it with https://github.com/Rui914/playeranim and PlayerSAO, but when I pass const std::string & the function returns only zeros. { x = 0, y = 0, z = 0 }. With std::string it returns the actual value.